intermediate_source/torchvision_tutorial.rst

```diff
@@ -343,178 +374,145 @@ expects during training and inference time on sample data.
-    Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.831
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.324
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.844
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.844
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.400
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.777
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.870
-   IoU metric: segm
-    Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.761
-    Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.969
-    Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.919
-    Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.341
-    Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.464
-    Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.788
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.303
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.799
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.799
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.400
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.769
-    Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.818
+    image = read_image("../_static/img/tv_tutorial/tv_image05.png")
+    eval_transform = get_transform(train=False)
 
-But what do the predictions look like? Let’s take one image in the
-dataset and verify
+    model.eval()
+    with torch.no_grad():
+        x = eval_transform(image)
+        # convert RGBA -> RGB and move to device
+        x = x[:3, ...].to(device)
+        predictions = model([x, ])
+        pred = predictions[0]
 
-.. image:: ../../_static/img/tv_tutorial/tv_image05.png
+    image = (255.0 * (image - image.min()) / (image.max() - image.min())).to(torch.uint8)
+    image = image[:3, ...]
+    pred_labels = [f"pedestrian: {score:.3f}" for label, score in zip(pred["labels"], pred["scores"])]
+    pred_boxes = pred["boxes"].long()
+    output_image = draw_bounding_boxes(image, pred_boxes, pred_labels, colors="red")
 
-The trained model predicts 9
-instances of person in this image, let’s see a couple of them:
+    masks = (pred["masks"] > 0.7).squeeze(1)
+    output_image = draw_segmentation_masks(output_image, masks, alpha=0.5, colors="blue")
 
-.. image:: ../../_static/img/tv_tutorial/tv_image06.png
+    plt.figure(figsize=(12, 12))
+    plt.imshow(output_image.permute(1, 2, 0))
 
-.. image:: ../../_static/img/tv_tutorial/tv_image07.png
 
-The results look pretty good!
+The results look good!
```

Comment:
Why remove the images? The rendered html now is:

![image](https://github.com/pytorch/tutorials/assets/1190450/65846981-eddd-4fa6-9eba-964bf133cba4)



which is unexpected